test/e2e/testdata/wolf-rbac/cmd.sh

```diff
@@ -20,20 +20,26 @@
 cd test/e2e/testdata/wolf-rbac/
 
 OPTION=$1
+COMPOSE="docker-compose"
+
+if  ! command -v "docker-compose" > /dev/null; then
+	echo "docker-compose not found, try docker compose"
+    COMPOSE="docker compose"
+fi
 
 if  [ $OPTION = "ip" ]; then
     echo -n `docker inspect -f '{{range .NetworkSettings.Networks}}{{.Gateway}}{{end}}' wolf-server`
 elif [ $OPTION = "start" ]; then
-    docker-compose -f 'docker-compose.yaml'  -p 'wolf-rbac' down
+    eval "$COMPOSE -f 'docker-compose.yaml'  -p 'wolf-rbac' down"
```

Comment:
We don't need "eval"

Reply:
Due to  the `compose` is a subcommand of `docker`, a form like this would be wrong, 

```
cmd='docker compose'
$cmd xxx
```

e.g.

```
~                                                                                                                     ⍉
▶ cm='docker compose'

~
▶ $cm version
zsh: command not found: docker compose

~                                                                                                                     ⍉
▶ eval $cm version
Docker Compose version v2.7.0

```

Reply:
If we don't use eval, is there a better way to do it?

Reply:
@Chever-John What do you think?

Reply:
```bash
cmd=( docker compose )
```

Reply:
> ```shell
> cmd=( docker compose )
> 
> $cmd
> ```

This method doesn't work. I can't think of any other way than eval.

<img width="663" alt="image" src="https://user-images.githubusercontent.com/38528079/185095182-46fc76ae-5e62-464e-b113-9cbf9b53a29a.png">